doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst

```diff
@@ -50,92 +45,50 @@ Procedure
-         salt-call state.apply components.system.network.mgmt.public
-
-   - For Data Network static IP, run the following command:
-
-      ::
-      
-         # Set Data Network
-         provisioner pillar_set "cluster/srvnode-1/network/data/public_ip" \"<IP address for public network>\"
-         provisioner pillar_set "cluster/srvnode-1/network/data/netmask" \"<Netmask for public data network>\"
-         salt-call state.apply components.system.network.data.public
-
-   **Note:** To verify the static IPs are configured, run the following command:
-
-      ::
-
-         cat /etc/sysconfig/network-scripts/ifcfg-ens32 |grep -Ei "ip|netmask|gateway"
-         cat /etc/sysconfig/network-scripts/ifcfg-ens33 |grep -Ei "ip|netmask|gateway"
-
-#. To check the CORTX cluster status, run the following command:
+   ::   
    
-   ::
+     vi ~/config.ini
+     
+#. Paste the code into the config file replacing your network interface names with ens32,ens33, ens34, and storage disks with partitions created in step 3:
    
-      pcs status
+   **Note:** The values used in `config.ini <https://github.com/seagate/cortx/blob/main/doc/ova/2.0.0/config.ini>`_ are for example purpose, update the values as per your environment.
    
-   **Note:** If the cluster is not running then stop and start cluster once using the following command:
-      
-      ::
-
-         cortx cluster stop
-         cortx cluster start
-
-   **Note** For VirtualBox users, you need to check if the CORTX hare cluster is running using the following command:
-
-      ::
-
-         hctl status
-
-      If the cluster is not running, start the cluster using the following command:
-
-      ::
-
-         hctl start
-
-
 #. Run **ip a l** and record the IP addresses of the following interfaces:
 
    * ens32 - Management IP: To access the CORTX GUI.
    * ens33 - Public data IP: To access S3 endpoint and perform IO operations.
    * ens34 - Private data IP: To perform CORTX internal communication.
 
    .. image:: https://github.com/Seagate/cortx/blob/main/doc/images/104networks.png
-
    
-#. Use the management IP from the **ip a l** command and configure the CORTX GUI, See `configure the CORTX GUI document <https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst>`_. 
-
-#. Run the following command and verify the S3 authserver and HA proxy are active and running:
+#. Create and run the reconfigure_network.sh script to ensure all the necessary services are operational,
 
    ::
+     
+     curl -O https://raw.githubusercontent.com/seagate/cortx/main/doc/ova/2.0.0/reconfigure_network.sh
+     chmod +x ./reconfigure_network.sh
+     ./reconfigure_network.sh
+     
+#. Reboot node
+     
+#. Run the following command to start the CORTX cluster:
 
-      systemctl status s3authserver
-      systemctl status haproxy
+    ::
+    
+     cortx cluster stop
```

Comment:
till this point, cluster is not running, this command failed

Reply:
corrected